website/src/pages/api/v0/announcements.ts

```diff
@@ -12,17 +12,17 @@ export default async function handle (req: NextApiRequest, res: NextApiResponse<
   res.json([
     {
       lang: {
-        en: 'There is currently a high DEX stabilization fee imposed on DUSD-DFI swaps due to DFIP 2206-D.',
-        de: 'Derzeit wird eine hohe DEX-Stabilisierungsgebühr bei Tausch von DUSD-DFI aufgrund von DFIP 2206-D erhoben.',
-        'zh-Hans': '由于社区提案 DFIP 2206-D，目前对 DUSD - DFI 兑换征收高额的 Dex 稳定费。',
-        'zh-Hant': '由於社區提案 DFIP 2206-D，目前對 DUSD - DFI 兌換徵收高額的 Dex 穩定費。',
-        fr: 'Il existe actuellement une taxe de stabilisation du DEX élevée imposée sur les échanges DUSD-DFI en raison du DFIP 2206-D.',
-        es: 'There is currently a high DEX Stabilization fee imposed on DUSD-DFI swaps due to DFIP 2206-D.',
-        it: 'There is currently a high DEX Stabilization fee imposed on DUSD-DFI swaps due to DFIP 2206-D.'
+        en: 'There is currently a high DEX stabilization fee imposed on DUSD-DFI, DUSD-USDT, and DUSD-USDC swaps due to DFIP 2206-D and DFIP 2207-B.',
```

Comment:
@thedoublejay please take note that once this is merged, the announcements on portfolio page will be updated in prod

Reply:
@pierregee can we have a different announcement for next release? Update the version of this one for older releases

Reply:
@thedoublejay kept the old one to display on version <=1.16.0 in LW. As for Scan, we don't have any versioning in the code so I updated it directly